ci/change-runtime-settings.py

```diff
@@ -0,0 +1,72 @@
+# coding=utf-8
+
+# Purpose:
+#   change runtime settings for build android and ios package
+
+import os
+import sys
+import configparser
+from collections import OrderedDict
+
+
+class MultiOrderedDict(OrderedDict):
+    def __setitem__(self, key, value):
+        if isinstance(value, list) and key in self and key.startswith('+'):
+            self[key].extend(value)
+        else:
+            super(MultiOrderedDict, self).__setitem__(key, value)
+
+class CustomInterpolation(configparser.BasicInterpolation):
+    def before_write(self, parser, section, option, value):
+        if option.startswith('+'):
+            return "\n{}=".format(option).join(value.split('\n'))
+        return value
+
+class CustomWriter:
+    output_file = None
+    def __init__(self, new_output_file):
+        self.output_file = new_output_file
+    def write(self, what):
+        self.output_file.write(what.replace('\n\t','\n'))
+        
+
+# modify runtime settings before cook
+def change_runtime_settings(project_home):
+    defaultengine = os.path.join(project_home, 'Game', 'Config', 'DefaultEngine.ini')
```

Comment:
I would like it if we could keep a coding style according to https://www.python.org/dev/peps/pep-0008/ and use `function_names` and `variable_names` where two words are used. Should be a quick and easy refactor to help with consistency. 
```suggestion
    default_engine_ini = os.path.join(project_home, 'Game', 'Config', 'DefaultEngine.ini')
```